Davidster/SOEN341Project
# Updated file structure and added some initial source files

Hey so I added somewhat of a skeleton to the project. From this starting point, we can add content to index.html or we can make new html files (loginPage.html, groupPage.html) which you guys can start filling up with some code if you want.

### To test the basic Hello World page I added:

1. `git pull` - sync up ur own computer with what's up on the repo
2. `git checkout buildProjectSkeleton` - check out contents of this pull request's branch
3. Go into your file explorer and open index.html in any browser
